rootfs/docker-entrypoint-initdb.d/001_setup_envdir.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+cd "$WALE_ENVDIR"
+
+# access-key-id and access-secret-key files are mounted in via kubernetes secrets
+AWS_ACCESS_KEY_ID=$(cat access-key-id)
+AWS_SECRET_ACCESS_KEY=$(cat access-secret-key)
+
+# setup envvars for wal-e
+cp access-key-id AWS_ACCESS_KEY_ID
+cp access-secret-key AWS_SECRET_ACCESS_KEY
```

Comment:
I couldn't seem to mount and rename individual secret files in kubernetes, so I opted for this route instead. This makes it so when we run `envdir $WALE_ENVDIR` in the container, `$AWS_ACCESS_KEY` and others are set correctly.